crates/ruff_linter/src/rules/ruff/rules/mutable_fromkeys_value.rs

```diff
@@ -73,13 +73,7 @@ pub(crate) fn mutable_fromkeys_value(checker: &mut Checker, call: &ast::ExprCall
     if attr != "fromkeys" {
         return;
     }
-    let Some(name_expr) = value.as_name_expr() else {
-        return;
-    };
-    if name_expr.id != "dict" {
-        return;
-    }
-    if !checker.semantic().is_builtin("dict") {
```

Comment:
Is it intentional that `is_builtin` uses `lookup_symbol` but the new implementation now uses `resolve_qualified_name` (which also works for member expression which the old implementation didn't)?

Reply:
I _don't_ think there's a need for the new abstraction to work with member expressions. I just used `resolve_qualified_name` because it seemed like the simplest way to implement the new abstraction. Do you think using `lookup_symbol` would be more efficient here? (_Goes to compare the two definitions..._)

Reply:
I'm not sure how we would be able to use `lookup_symbol` here, as `lookup_symbol` takes `&str` (representing the name of the symbol) rather than an AST node. And we don't know the name of the symbol -- even if it's a builtin, as you can do `from builtins import open as o` -- unless we resolve the qualified name